.github/workflows/tests.yml

```diff
@@ -80,7 +84,7 @@ jobs:
         env:
           ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
           ECR_REPOSITORY: kilt/prototype-chain
-          IMAGE_TAG: latest-develop
+          IMAGE_TAG: ${{ matrix.image }}
         run: |
           yarn install --immutable
           yarn build
```

Comment:
building twice feels completely unnecessary, but as far as I can see there is no matrix strategy for a single step only - so I guess there is no way around it. But it's another argument to finally figure out a way to build once and use the wherever required (running example, integration testing, maybe even unit testing ?)